README.md

```diff
@@ -62,24 +62,23 @@ Issues labeled [help wanted](https://github.com/photoprism/photoprism/labels/hel
 [easy](https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy) can be
 good (first) contributions. 
 
-Follow us on [Twitter][twitter] to receive regular project updates and discuss development 
-related topics. Don't be afraid to ask stupid questions.
+Follow us on [Twitter][twitter] to receive regular project updates and discuss development related topics. Don't be afraid to ask stupid questions.
 
 ## Funding ##
 
 This software is about freedom and privacy but not necessarily about free beer. We feel like it
-would be a mistake to state there will be no costs, because clearly we have huge expenses, your server hardware
+would be a mistake to state there will be no costs because clearly, we have huge expenses, your server hardware
 will have a price tag, and then maybe you'd like to have some extra features that need to be developed.
 
 You're welcome to support us via [GitHub Sponsors](https://github.com/sponsors/photoprism), 
-especially if you have feature requests, or need help with using our software.
+especially if you have feature requests or need help with using our software.
 In addition, you can find us on [Patreon](https://www.patreon.com/photoprism)
 and [PayPal](https://www.paypal.me/photoprism). 
 
 Your continuous support helps...
 
-* to pay for operating expenses and external services like satellite maps
-* developing new features, and keeping them free for everyone 🌈
+* pay for operating expenses and external services like satellite maps
+* developing new features and keeping them free for everyone 🌈
```

Comment:
```suggestion
* develop new features and keep them free for everyone 🌈
```

Reply:
Isn't developing a continuous process that requires the present participle?

Reply:
You would read this combined with the part before the list "Your continuous support helps develop new features and keep them free for everyone" vs "Your continuous support helps developing new features and keeping them free for everyone"

Reply:
We only pay our bills once a month, so not the same :wink: